src/SFA.DAS.Data.Functions/ProcessTransferRelationshipStartMessage.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.ServiceBus.Messaging;
+
+namespace SFA.DAS.Data.Functions
+{
+    public class ProcessTransferRelationshipStartMessage
+    {
+        [FunctionName("ProcessTransferRelationshipStartMessage")]
+        public static void Run([ServiceBusTrigger("transfer_connection_invitation_sent", "RDS_TransferConnectionInvitationSentProcessor", AccessRights.Manage,Connection = "MessageBusConnectionString")]string paymentCreatedMessage, ExecutionContext executionContext, TraceWriter log)
```

Comment:
Is this actually doing anything?  I may be being a bit thick!

Reply:
No, it's not doing anything. It's Steve's legacy and any changes to it here are purely to fix the failing build. There is a proper update and fix coming in another branch that reads transfer relationships from EAS queue

Reply:
Excellent, thought it would be something like that.